README.md

```
@@ -2,313 +2,1370 @@
-* [Isitha Subasinghe](https://isub.dev)
-* [Ismail Ghallou aka Smakosh](https://smakosh.com)
+
+- [Ian Lunn](https://ianlunn.co.uk) [Frontend Developer | Responsive Web Specialist]
+- [Ibrahim Hizlioglu](https://www.ibrahimhizlioglu.com)
+- [Ike Ofoegbu](https://iodev.io)
+- [Ilija Korodić](https://ilijakorodic.netlify.app) [Frontend Developer]
+- [Ilke Ozsekerli](https://ilkeozs.info)
+- [Ilya Dyomin](https://sskd.tech) [Front-End Developer]
+- [Ilyes Landolsi](https://ilandols.com)
+- [Indrajeet Nikam](https://indrajeet.me)
+- [Indrayudh Dhara](https://my-portfolio-ba1h.vercel.app)[Full Stack Developer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://my-portfolio-ba1h.vercel.app)